csharp/Ice/hello/Hello.ice

```diff
@@ -8,7 +8,7 @@ module Demo
 {
     interface Hello
     {
-        idempotent void sayHello(int delay);
-        void shutdown();
+        void sayHello();
+        string recieveResponse();
```

Comment:
receive is spelled `receive`. And naturally we should not have such an operation. sayHello should return a string - the response - not void. `sayHello` should also take a parameter, like:

```
string sayHello(string greeting);
```